escape-room/doc/glossary.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Virtuelle Escape-Rooms in der Lehre - Glossar"
+---
+
+# Akteure
+
+## Designer
+
+Designer erstellen mit dem Dungeon Escape-Rooms zur Erreichung von Zielen aus der Lehre.
+
+## Spieler
+
+## (Spieler-) Charakter
+
+Charaktere sind von Spielern gesteuerte Figuren.
+
+## NPCs
+
+NPCs sind Figuren, die von der Simulation gesteuert werden.
+
+# Geschichte
+
+Die Geschichte ist das, was die Spieler bei dem Durchlauf eines Escape-Rooms erleben.
+
+## Geschehnis
+
+Geschehnisse sind Zustandsänderungen der Geschichte, deren Zeitpunkt und Ablauf vom Designer vorgegeben ist.
+
+> Bsp.: Ein Dschinn erscheint 30 Sekunden nach Spielstart und verlangt von den Charakteren die Preisgabe ihrer Namen.
+
+## Ereignis
+
+Ereignisse werden von Spielern initiiert bringen die Handlung der Geschichte/die Beendigung des Escape-Rooms voran.
+
+TODO "geschehnis" und "ereignis" in anlehnung an die erzähltheorie von lahn und meister. das ereignis ist wichtigere. geschehnisse sind nicht wesentlich.
+
+### (Lern-) Rätsel
+
+Rätsel sind Ereignisse, die auf die Erreichung von Lernzielen zusteuern. Lernziele sind z.B. Inhalte von Lehrveranstaltungen oder Soft-Skills, wie logisches Denken, Problemlösungsstrategien, Kommunikation in der Gruppe und Zusammenarbeit.
+
+TODO wollen wir Rätsel für Soft-Skills namentlich trennen?
```

Comment:
ich kann mir grad keinen grund dafür vorstellen. 

aus sicht der lehrenden habe ich lernziele (outcomes), die ich mit aufgaben adressieren will. vielleicht ist das ein begriff, der hier mit eine rolle spielt. aber vielleicht auch nicht - es schreibt ja niemand an die aufgaben im dungeon dran, was das lernziel ist? oder doch? hmm.

Reply:
mir ist das auch nicht ganz klar. deswegen nur "frage im TODO".

mein gedanke war folgender:
als lehrende person denkt man vermutlich eher vordergründig an die eigenen lerninhalte. dann wäre es praktisch, wenn einem die soft-skills in der nähe von dem "füllmaterial" (es gibt eine kategorie von nacharbeiten, die wahrscheinlich im vorgehen lannden) angeboten werden oder implizit in der implementierung von den (kern-)rätsel und der "escape-room-logic" sind.

wahrscheinlich ist es einfacher darüber zu entscheiden, wenn andré auch ein paar gedanken an das (leitende) vorgehen verschwendet hat.

kannst du sonst kurz erfahrungswerte teilen, wie und wann soft-skills in deine unterrichtsplanung einfließen? (du bist ja der eine primäre dungeon-lehrer und die literatur sind andere)